### PR TITLE
Fix multiplying TextureBrush with a disposed matrix on Unix

### PR DESCRIPTION
**Pull request by hughbe**
Also fixed in Libgdiplus here: https://github.com/mono/libgdiplus/pull/80

Also enabling a test that should work

### CODE REVIEW

**Comment by mellinoe** [comment]
@MattGal It looks like `libgdiplus` isn't being found on OSX 10.12 and 10.13. Was it installed into the machines used for these test runs?

File: src/System.Drawing.Common/tests/Helpers.cs [hunk 1]
```diff
@@ -23,13 +23,14 @@ public static bool GetGdiplusIsAvailable()
             }
             else
             {
+                return true;/*
```
Comment (mellinoe) [inline src/System.Drawing.Common/tests/Helpers.cs]: Can we remove this before merging?

Comment (hughbe) [reply src/System.Drawing.Common/tests/Helpers.cs]: Oof, sorry

**Comment by MattGal** [comment]
Fixing the OSX .Open machines is tracked by https://github.com/dotnet/core-eng/issues/1792

**Pull request merged**